examples/java/pom.xml

```diff
@@ -294,6 +316,30 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-runners-flink_2.10</artifactId>
+      <version>${project.version}</version>
```

Comment:
These should all be runtime dependencies. I believe the Java 8 examples have a more hygienic configuration.